File: tests/query.js
function sumArr(arr) {
	var res = 0
	for (var i = 0; i < arr.length; i++) {
		res += arr[i]
	}

	return res
}

var sum = 0
function scan(open, high, low, close, volume) {
	sum += sumArr(open);
	sum += sumArr(high);
	sum += sumArr(low);
	sum += sumArr(close);
	sum += volume.length;

	return sum;
}

File: tests/query.jl
total = Float32(0)
function scan(
	open::Vector{Float32},
	high::Vector{Float32},
	low::Vector{Float32},
	close::Vector{Float32},
	volume::Vector{UInt64}
)
	global total += sum(open)
	global total += sum(high)
	global total += sum(low)
	global total += sum(close)
	global total += sum(volume)

	total
end
